Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jamulus/-/1-index/horizontal-auto.svg)

ignotus666 commented on _translator-files/po/nb_NO/Client-Troubleshooting.po:
```diff
@@ -31,7 +32,7 @@ msgstr "wiki"
 #, no-wrap
 msgid "/wiki/Client-Troubleshooting"
-msgstr ""
+msgstr "/wiki/klientproblemløsing"
```
@comradekingu these links shouldn't be translated.

comradekingu commented:
@ignotus666 OK. I suppose flagging those as "read-only" in WL helps avoid the mistake.
https://hosted.weblate.org/user/kingu/ is not an admin, so I can't fix that other than to undo the translations.

jujudusud commented:
@ignotus666 :
Can you please answer this question?
https://hosted.weblate.org/translate/jamulus/unregistered-servers/fr/?q=has:comment#comments

jujudusud commented:
French Part of the website translation is ready to be merged.

ignotus666 commented:
@jujudusud answered!

jujudusud commented:
Thanks a lot @ignotus666

henkdegroot commented:
All translation for Dutch is included in this PR. Ready to be merged.

trebmuh commented:
I had time to review and fix the website changes.
It's all good for French now.

ann0see commented on _translator-files/po/de/Running-a-Server.po:
```diff
@@ -254,9 +255,8 @@ msgstr "2. Unregistriert"
 #: ../wiki/en/Running-a-Server.md:87
-#, fuzzy
 msgid "This is the default when starting a Server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your Server's address can connect to it."
-msgstr "Dies ist die Standardeinstellung, wenn du einen Server zum ersten Mal startest. Unregistrierte/private Server werden nicht in Verzeichnissen aufgeführt, so dass nur Musiker, die die Adresse deines Servers kennen, eine Verbindung zum Server herstellen können. Dies ist nützlich, da du in Jamulus nicht kontrollieren kannst, wer sich mit deinem Server verbindet."
```
This was correct German.

weblate committed:
Update German translation using Weblate

Currently translated at 30.1% (47 of 156 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: ann0see <20726856+ann0see@users.noreply.github.com>
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/de/
Translation: Jamulus/Navigation
Translation: Jamulus/Running-a-Server

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/
Translation: Jamulus/Navigation

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/
Translation: Jamulus/Navigation

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/
Translation: Jamulus/Navigation

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/
Translation: Jamulus/Navigation

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/
Translation: Jamulus/Navigation

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/navigation/
Translation: Jamulus/Navigation

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Running-a-Server
Translate-URL: https://hosted.weblate.org/projects/jamulus/running-a-server/

ann0see reviewed (approved):
German is ok too.
